io/net/bonding.py

```diff
@@ -274,8 +278,21 @@ def bond_setup(self, arg1, arg2):
             linux_modules.load_module("bonding")
             genio.write_file(self.bonding_masters_file, "+%s" % self.bond_name)
             genio.write_file("%s/bonding/mode" % self.bond_dir, arg2)
-            genio.write_file("%s/bonding/miimon" % self.bond_dir, "100")
-            genio.write_file("%s/bonding/fail_over_mac" % self.bond_dir, "2")
+            genio.write_file("%s/bonding/miimon" % self.bond_dir,
+                             self.miimon)
+            genio.write_file("%s/bonding/fail_over_mac" % self.bond_dir,
+                             self.fail_over_mac)
+            genio.write_file("%s/bonding/downdelay" % self.bond_dir,
+                             self.downdelay)
+            dict = {'0': ['packets_per_slave', 'resend_igmp'], '1': ['num_unsol_na', 'primary', 'primary_reselect', 'resend_igmp'],
+                    '2': ['xmit_hash_policy'], '4': ['lacp_rate', 'xmit_hash_policy'],
+                    '5': ['tlb_dynamic_lb', 'primary', 'primary_reselect', 'resend_igmp', 'xmit_hash_policy', 'lp_interval'],
+                    '6': ['primary', 'primary_reselect', 'resend_igmp', 'lp_interval']}
```

Comment:
@smruti77 please get this into pep8 standards.

Reply:
@smruti77 suggestion:
Can we also get miimon, failovermac, downdelay into this dictionary (for all modes), so that they are also taken care in for loop ?

Reply:
@narasimhan-v that three are for all the mode, why need to move to dictionary.and in dictionary their no mode3 so it should be better as it is.

Reply:
@smruti77 you can create a key for mode 3 and add these 3 params.
Right now, there are 2 types of params: common and mode specific. Mode specific you are handling via dictionary, I am suggesting to handle common params also similarly.